src/Standards/Finetic/ruleset.xml

```diff
@@ -143,6 +143,8 @@
     <rule ref="SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint"/>
     <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingAnyTypeHint"/>
     <rule ref="SlevomatCodingStandard.TypeHints.UselessConstantTypeHint"/>
+    <rule ref="SlevomatCodingStandard.Commenting.UselessFunctionDocComment"/>
```

Comment:
> This sniff can cause an error if you're overriding or implementing a parent method which does not have typehints. In such cases add @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint annotation to the method to have this sniff skip it.

Dit moeten we testen, want te lijkt me niet handig om een `phpcsSuppress` toe te voegen (of een `// phpcs:ignore SlevomatCodingStandard.Commenting.UselessFunctionDocComment` als de parent niet goed is.